app/controllers/workflows_controller.rb

```diff
@@ -33,28 +37,41 @@ def new
 
   def create
     skip_authorization
+    workflow_id = params[:workflow][:id]
 
-    workflow_hash = credential.accessible_workflow?(params[:workflow][:id])
+    workflow_hash = { id: workflow_id }
+    workflow_hash = credential.accessible_workflow?(params[:workflow][:id]) unless Rails.env.development?
 
     unless workflow_hash.present?
       skip_authorization
       head :forbidden
       return
     end
 
-    @workflow = Workflow.new(workflow_params.merge(id: params[:workflow][:id],
-                                                   project_id: workflow_hash["links"]["project"]))
+    panoptes_workflow = Effects.panoptes.workflow(workflow_id) unless Rails.env.development?
+    if Rails.env.development? || Rails.env.test?
+      panoptes_workflow = { workflow_id: workflow_id, project_id: workflow_id, display_name: 'New Workflow'}.stringify_keys
+    end
+
+    @workflow = Workflow.new(workflow_params.merge(
+      id: workflow_id,
+      project_id: panoptes_workflow["project_id"],
+      name: panoptes_workflow["display_name"]
+    ))
 
     @workflow.save
-    DescribeWorkflowWorker.perform_async(@workflow.id)
+    DescribeWorkflowWorker.perform_async(@workflow.id) unless @workflow.id.blank?
```

Comment:
Do we still need to run this worker if we're loading the display name now? Could we do it all in one go instead. I think with this PR we'd end up making 4 API calls to Panoptes, 3 to load the same workflow over and over, and one to load the project separately (which maybe could use includes?).

Reply:
I still need it to fetch the project name, but the other calls have been removed by referring to the workflow_hash